x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/user/UsernamesField.java

```diff
@@ -16,6 +16,8 @@ public final class UsernamesField {
     public static final String SYSTEM_ROLE = "_system";
     public static final String XPACK_SECURITY_NAME = "_xpack_security";
     public static final String XPACK_SECURITY_ROLE = "_xpack_security";
+    public static final String DLM_NAME = "_dlm";
+    public static final String DLM_ROLE = "_dlm";
```

Comment:
nitpick and outside the scope of this PR (i.e. feel free to ignore or consider a follow up).... IMO we should not list the role name here for any internal users since those names are just implementation details of internal users. IMO only reserved roles should listed here as they might be some case to reference them somewhere but the internal users roles should just be defined local to where the InternalUser is defined and possibly just a blank value.

Reply:
I'm ++ on this proposal. Will revisit in a follow up.